CSharpBasics2/CSharpBasics2/Program.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Xml.Linq;
+
+
+class Homework
+{
+    static void Main()
+    {
+       
+        {
+            // TASK_1
+
+            Console.WriteLine("Enter the number1:");
+            var numberToString = Console.ReadLine();
+            int q = Convert.ToInt32(numberToString);
+            double w = Convert.ToDouble(numberToString);
+            char r = Convert.ToChar(numberToString);
+            float t = Convert.ToSingle(numberToString);
+            Console.WriteLine($" {q}, {w}, {r}, {t}");
+            Console.ReadLine();
+
+        }
+
+        {
+            // TASK_2
+
+            Console.WriteLine("Enter the number2:");
+            var numberToString = Console.ReadLine();
+            int a = int.Parse(numberToString);
+            Console.WriteLine("Enter the number2:");
+            var numberToString1 = Console.ReadLine();
+            double b = double.Parse(numberToString);
+           
+
+        }
+
+        {
+            // TASK_3
+
+            Console.WriteLine("Enter the number3:");
+            var numberToString = Console.ReadLine();
+            int c;
+
+            var isNumber = int.TryParse(numberToString, out c);
+            Console.WriteLine($"Number is {c}");
+            Console.WriteLine("Success");
+
+        }
+
+        {
+            // TASK_4
+
+            Console.WriteLine("Enter the number4:");
+            var numberToString = Console.ReadLine();
+            int d;
+            var isNumber = int.TryParse(numberToString, out d);
+            int e = d + 3;
+            int f = e + d;
+            Console.WriteLine($" {d}, {e}, {f}");
+            Console.ReadLine();
+
+        }
+
+        {
+            // Task_5
+
+            string s1 = "Apples";
+            double w1 = 2.32;
+            string s2 = "Juice";
+            double w2 = 2.05;
+            string s3 = "Milk";
+            double w3 = 1.56;
+            string s4 = "Cucumber";
+            double w4 = 2;
+            string s5 = "Cake";
+            double w5 = 6;
+            string s6 = s1 + w1;
+            string s7 = s2 + w2;
+            string s8 = s3 + w3;
+            string s9 = s4 + w4;
+            string s10 = s5 + w5;
+            string s11 = "All";
+            double w6 = ((w1 + w2 + w3 + w4 + w5) - (w1 + w2 + w3 + w4 + w5)*0.15);

```

Comment:
please put the sum of all items in the separate variable, so you will be able to use it when counting discount without entering this line again